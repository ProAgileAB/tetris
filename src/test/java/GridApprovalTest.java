import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;


class GridApprovalTest {

    // # initial state - an empty grid
    // a grid top left cell set to a string
    // a grid bottom right cell set to number
    // a full grid

    @Test
    void anEmptyGrid() {
        var grid = new MyBoundedGrid<String>(2, 3);
        Approvals.verify(grid);
    }
}