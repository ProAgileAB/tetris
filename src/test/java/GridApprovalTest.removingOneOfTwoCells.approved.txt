A grid of 1 rows 2 columns. Content:
.B